experimentalServer/config.json

```diff
@@ -0,0 +1,5 @@
+{
+"Message": "If you would like to support this instance, consider donating!",
```

Comment:
Can you leave the `Message` field blank?

I don't expect anyone to donate to me for this project/hosting, so I would prefer if this was not the default value :)